hackerrank/src/Algorithms/Strings/TwoCharacters.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace Fedotkin.Dotnet.HackerRank.Algorithms.Strings.TwoCharacters;
+
+public class Result
+{
+    private static int AlternationCheck(string s, char symb1, char symb2)
+    {
+        var checkList = new List<char>(); // List of two given characters
+
+        for (int i = 0; i < s.Length; i++) // Filling the list
+        {
+            if (s[i] == symb1 || s[i] == symb2)
+                checkList.Add(s[i]);
+        }
+
+        char first = checkList[0], second = checkList[1];
+
+        for (int i = 2; i < checkList.Count; i++) // Check for alternation
+        {
+            if ((i % 2 == 0 && checkList[i] != first) | (i % 2 == 1 && checkList[i] != second))
+                return 0;
+        }
+
+        return checkList.Count;
+    }
+
+    public static int Alternate(string s)
+    {
+        if (string.IsNullOrEmpty(s))
+            return 0;
+
+        int result = 0;
+        var symbols = new List<char>(); // General list of all existing symbols in string "s"
+
+        foreach (char symbol in s) // Filling the list
+        {
+            if (!symbols.Contains(symbol))
+                symbols.Add(symbol);
+        }
```

Comment:
Вышэй: 
`using System.Linq`
і ніжэй:
```suggestion
        var symbols = s.Distinct().ToList(); // General list of all existing symbols in string "s"
```